action.yaml

```diff
@@ -1,26 +1,74 @@
 name: "Generate SPEC-0000 Data"
 description: "Based on the current SPEC 0 schedule, generate a tarball with the latest versions of all packages."
+author: Sam Vente
```

Comment:
Do have `Scientific Python Developers` as author here

See policies for our tools at the bottom: https://tools.scientific-python.org/